pkg/minishift/registration/utils.go

```diff
@@ -30,7 +31,7 @@ type RegistrationParameters struct {
 }
 
 // Register host VM
-func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+func RegisterHostVM(host *host.Host, param *RegistrationParameters, proxyInfo string) error {
```

Comment:
Yes it is, this is just the portion passed to the registration algorithm. You would still keep the OpenShift and Docker config, but what you need for the registration part gets passed via RegistrationParameters.